api/agent.go

```diff
@@ -363,11 +363,12 @@ func NewControllerWrapper(config *loadtest.Config, controllerConfig interface{},
 			Password:     password,
 		}
 		store, err := memstore.New(&memstore.Config{
-			MaxStoredPosts:          500,
-			MaxStoredUsers:          1000,
-			MaxStoredChannelMembers: 1000,
-			MaxStoredStatuses:       1000,
-			MaxStoredThreads:        500,
+			MaxStoredPosts:          250,
+			MaxStoredUsers:          500,
+			MaxStoredChannelMembers: 500,
+			MaxStoredStatuses:       500,
+			MaxStoredThreads:        250,
+			MaxStoredReactions:      10,
```

Comment:
I know it's a bit more work but I'd really like to see these exposed through the agent's config. We currently have a use case of generating the data but real clients don't really have limits to how many entities they can store so from a simulation quality perspective it would be best to keep these higher when possible.

Reply:
>  but real clients don't really have limits to how many entities they can store so from a simulation quality perspective it would be best to keep these higher when possible.

It'll be a tradeoff essentially. In real life, the webapp can accumulate upto 1GiB of mem. (It's showing 500MiB right now). That would be very very costly in terms of resources. And I think that was one of the reason we didn't choose to go the route of headless browser emulation.

But I agree that it will be good to have them exposed via agent's config. Created: https://mattermost.atlassian.net/browse/MM-52951